tests/generation/FileGlobs.py

```diff
@@ -100,15 +100,21 @@ def test_get_project_files_gitignore(self):
 
     def test_get_project_files_ci_mode(self):
         orig_cwd = os.getcwd()
-        os.chdir(os.path.dirname(os.path.realpath(__file__)) + 
+        os.chdir(os.path.dirname(os.path.realpath(__file__)) +
             os.sep + "file_globs_ci_testfiles")
 
         with suppress_stdout():
             res, _ = get_project_files(self.log_printer, self.printer, os.getcwd()
                 , True)
-            self.assertIn(os.path.join(os.getcwd(), "src", "file.c"), res)
-            self.assertIn(os.path.join(os.getcwd(), "root.c"), res)
-            self.assertIn(os.path.join(os.getcwd(), "ignore_dir", "src.c"), res)
-            self.assertIn(os.path.join(os.getcwd(), "ignore_dir", "src.js"), res)
+
+            paths = [
+                os.path.join(os.getcwd(), "src", "file.c"),
+                os.path.join(os.getcwd(), "root.c"),
+                os.path.join(os.getcwd(), "ignore_dir", "src.c"),
+                os.path.join(os.getcwd(), "ignore_dir", "src.js")
```

Comment:
add trailing comma so a new item can be added to the list without modifying this line.